update_version.py

```diff
@@ -6,6 +6,8 @@
 
 import sys
 import re
+import toml
```

Comment:
This file (update_version.py) might not be needed if using regex also in pyproject.toml